### PR TITLE
4.2 update ova documentation

### PR DESCRIPTION
**Pull request by rauldpm**
Related: https://github.com/wazuh/wazuh-packages/issues/799

## Description

This PR adds the necessary changes according to the new OVA model

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [ ] Used uppercase only on nouns. 
- [ ] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

image
image
image

### CODE REVIEW

File: source/development/packaging/generate-ova.rst [hunk 1]
```diff
@@ -32,33 +32,29 @@ Execute the ``generate_ova.sh`` script, with the different options you desire.
...
-    -o, --opendistro       [Required] Version of Open Distro for Elasticsearch.
-    -f, --filebeat         [Required] Filebeat's version.
-    -r, --repository       [Required] Select the software repository [prod/dev].
-    -b, --branch           [Optional] Branch/tag of the Wazuh repository.
-    -s, --store <path>     [Optional] Set the destination absolute path of package.
-    -c, --checksum <path>  [Optional] Generate checksum.
-    -u, --ui-revision      [Optional] Revision of the UI package. By default, 1.
-    -h, --help             [  Util  ] Show this help.
+  General usage: generate_ova.sh [OPTIONS]
+    -w,    --wazuh            [Optional] Select the wazuh major version [4.1, 4.2]. By default: 4.2
```
Comment (s-ocando) [inline source/development/packaging/generate-ova.rst:36]:
```suggestion
    -w,    --wazuh            [Optional] Select the Wazuh major version [4.1, 4.2]. By default: 4.2
```

File: source/development/packaging/generate-ova.rst [hunk 2]
```diff
@@ -32,33 +32,29 @@ Execute the ``generate_ova.sh`` script, with the different options you desire.
...
+    -r,    --repository       [Optional] Select the software repository [prod/dev]. By default: prod
+    -s,    --store <path>     [Optional] Set the destination absolute path where the ova file will be stored.
```
Comment (s-ocando) [inline source/development/packaging/generate-ova.rst:38]:
```suggestion
    -s,    --store <path>     [Optional] Set the destination absolute path where the OVA file will be stored.
```

File: source/development/packaging/generate-ova.rst [hunk 3]
```diff
@@ -32,33 +32,29 @@ Execute the ``generate_ova.sh`` script, with the different options you desire.
...
+    -c,    --checksum         [Optional] Generate checksum [yes/no]. By default: no
+    -g,    --debug            [Optional] Set debug mode on [yes/no]. By default: no
+    -h,    --help             [  Util  ] Show this help.
 
 The options for the repository indicates whether the packages used to install Wazuh are the production ones or not.
 
- * Stable: The OVA uses released packages.
- * Unstable: The OVA uses not released packages.
+ * prod: The OVA uses released packages.
+ * dev: The OVA uses not released packages.
```
Comment (s-ocando) [inline source/development/packaging/generate-ova.rst:46]:
```suggestion
 * dev: The OVA uses pre-release packages.
```

Comment (rauldpm) [reply source/development/packaging/generate-ova.rst]: Fixed in https://github.com/wazuh/wazuh-documentation/pull/4150/commits/9df2633702e37b1a3c1c1642d89251a4097ec848

**Review by s-ocando — APPROVED** [review]
LGTM

**Pull request merged**